cmd/manager/main.go

```diff
@@ -47,52 +57,74 @@ func printVersion() {
 	logrus.Printf("operator-sdk Version: %v", sdkVersion.Version)
 }
 
+func setupScheme() *kruntime.Scheme {
+	scheme := kruntime.NewScheme()
+
+	utilruntime.Must(apis.AddToScheme(scheme))
+	utilruntime.Must(olmv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(v1beta1.AddToScheme(scheme))
+	utilruntime.Must(corev1.AddToScheme(scheme))
+
+	if configv1.IsAPIAvailable() {
+		utilruntime.Must(apiconfigv1.AddToScheme(scheme))
+	}
+
+	return scheme
+}
+
 func main() {
 	printVersion()
 
 	var (
-		clusterOperatorName string
-		tlsKeyPath          string
-		tlsCertPath         string
-		version             bool
+		clusterOperatorName     string
+		tlsKeyPath              string
+		tlsCertPath             string
+		leaderElectionNamespace string
+		version                 bool
 	)
-	flag.StringVar(&clusterOperatorName, "clusterOperatorName", "", "the name of the OpenShift ClusterOperator that should reflect this operator's status, or the empty string to disable ClusterOperator updates")
-	flag.StringVar(&defaults.Dir, "defaultsDir", "", "the directory where the default CatalogSources are stored")
+	flag.StringVar(&clusterOperatorName, "clusterOperatorName", "", "configures the name of the OpenShift ClusterOperator that should reflect this operator's status, or the empty string to disable ClusterOperator updates")
+	flag.StringVar(&defaults.Dir, "defaultsDir", "", "configures the directory where the default CatalogSources are stored")
 	flag.BoolVar(&version, "version", false, "displays marketplace source commit info.")
 	flag.StringVar(&tlsKeyPath, "tls-key", "", "Path to use for private key (requires tls-cert)")
 	flag.StringVar(&tlsCertPath, "tls-cert", "", "Path to use for certificate (requires tls-key)")
+	flag.StringVar(&leaderElectionNamespace, "leader-namespace", "openshift-marketplace", "configures the namespace that will contain the leader election lock")
```

Comment:
Another quick note - this was namely added to accommodate the hypershift use case, where you may be housing locks outside of the namespace the actual operator deployments are being run.